portfolio/src/main/webapp/index.html

```diff
@@ -49,7 +49,9 @@ <h2>Comments</h2>
                 <input type="submit" class="submit-button">
 
             <ul id="comment-container"></ul><br>
+            <button type="button" class="submit-button" onclick="deleteComments()">Delete all comments</button> 
         </div>
+
```

Comment:
Was this extra newline intended?

Reply:
I originally had it there to further separate the about&comment section from the next but I changed it back to a single newline for style consistency:)